zipline/pipeline/loaders/blaze/estimates.py

```diff
@@ -103,11 +106,21 @@ def __init__(self,
         self._data_query_time = data_query_time
         self._data_query_tz = data_query_tz
         self._checkpoints = checkpoints
+        self._split_adjustments = split_adjustments_loader
+        self._split_adjusted_column_names = split_adjusted_column_names
+        self._split_adjusted_asof = split_adjusted_asof
 
     def load_adjusted_array(self, columns, dates, assets, mask):
         # Only load requested columns.
         requested_column_names = [self._columns[column.name]
                                   for column in columns]
+        requested_spilt_adjusted_columns = None
+        if self._split_adjusted_column_names:
+            requested_spilt_adjusted_columns = [
+                column_name
+                for column_name in self._split_adjusted_column_names
+                if column_name in requested_column_names
+            ]
```

Comment:
Yes, it's a valid value, as long as `split_adjustments_loader` and `split_adjusted_asof` are also both `None`. That just means that no split adjustments should be applied to the given data. However, it's not valid for only a subset of those 3 fields to have a value of `None`. The method `validate_split_adjustments_args` in `pipeline.loaders.earnings_estimates.EarningsEstimatesLoader` checks for those cases.